tests/unit/sklearn/test_pipeline.py

```diff
@@ -75,3 +75,59 @@ def test_score_logs_metric(project_client, fake_estimator_cls):
             pipeline.score(["fake data"])
 
     mock_log_metric.assert_called_once()
+
+
+def test_fit_multiple_scores(project_client, fake_estimator_cls):
+    project = project_client
+    estimator = fake_estimator_cls()
+    steps = [("est", estimator)]
+    pipeline = RubiconPipeline(project, steps)
+
+    with patch.object(Pipeline, "score", return_value=None):
+        with patch.object(EstimatorLogger, "log_metric", return_value=None):
```

Comment:
for each patched logger, add `as mock_logger` and assert `mock_logger` is called once (or however many times it should be)